src/cdk/stepper/stepper.md

```diff
@@ -0,0 +1,59 @@
+CDK stepper provides a foundation upon which more concrete stepper varities can be built upon. A
+stepper is a wizard-like workflow that divides content into logical steps
+
+### Behavior captured by CdkStepper
+The base CDK version of the stepper primarily manages which step is active. This includes handling
+keyboard interactions and exposing an API for advancing or rewinding through th workflow.
+
+#### Linear stepper
+A stepper marked as `linear` requires the user to complete previous steps before proceeding
+to following steps. For each step, the `stepControl` attribute can be set to the top level
```

Comment:
I think the `to following steps` is redundant.

Reply:
Done